docs/src/index.md

```diff
@@ -24,13 +30,12 @@ Pkg.add("https://github.com/NumericalEarth/Breeze.jl.git")
 
 ## Quick Start
 
-A basic free convection simulation:
+A basic free convection simulation with `AtmosphereModel`:
 
 ```@example
-using Oceananigans
+using Breeze
 using Oceananigans.Units
```

Comment:
I'm wondering if it makes sense to somehow make Breeze re-export some things. E.g., would it make sense if here we could just write:

```suggestion
using Breeze.Units
```

cc @giordano

Reply:
Honestly I'm not a fan of that, then you're tying the API of one package A to another package B, with unintended consequences, like code breaking when B gets upgraded without any changes in A